src/config/xorg/index.md

```diff
@@ -1,57 +1,50 @@
 # Xorg
 
-## Drivers
-
-## Display managers
-
-## Session and seat management
+This section details the manual installation and configuration of the Xorg
+display server and common related services and utilities. If you would just like
+to install a full desktop environment, it is recommended to try one of the
+[flavor images](../../installation/live-images/index.md#flavor-images)
 
-Session and seat management is not necessary for every setup, it is used to
-provide device access on the fly for the currently active user session.
+## Installation
 
-For desktop environments like Gnome [elogind](#elogind) is necessary.
+Void provides a comprehensive `xorg` package which installs the server and all
+of the free video drivers, fonts, and base applications. This package is a safe
+option, and should be adequate for most systems which don't require proprietary
+video drivers.
 
-### ConsoleKit2
+If you would like to select only the packages you need, the `xorg-minimal`
+package contains the base xorg server *only*. Remember that you will likely need
+to install a font package (like `xorg-fonts`), a terminal emulator (like
+`xterm`), and a window manager to have a usable graphics system.
 
-Install ConsoleKit2 and activate its service and make sure the dbus and the
-cgmanager services are activated too.
-
-```
-# xbps-install -S ConsoleKit2
-# ln -s /etc/sv/dbus /var/service/
-# ln -s /etc/sv/cgmanager /var/service/
-# ln -s /etc/sv/consolekit /var/service/
-```
+## Drivers
 
-If you don't use a display manager or your display manager doesn't start a
-ConsoleKit2 session on its own you need to start a ConsoleKit2 session from your
-`.xinitrc`. ConsoleKit2 comes with a `xinitrc.d` script
-(`/etc/X11/xinit/xinitrc.d/90-consolekit`) which sets the `STARTUP` variable to
-the appropriate way to start the session.
+### Open Source Drivers
 
-The following `.xinitrc` script sources all scripts in
-`/etc/X11/xinit/xinitrc.d` and starts the window manager of your choice with a
-session.
+The open source drivers are installed with the `xorg` package by default, or may
+be installed individually.
 
-```
-#!/bin/sh
-#
-# ~/.xinitrc
-#
-# Executed by startx (run your window manager from here)
+| Brand  | Type        | Driver               |
+|--------|-------------|----------------------|
+| AMD    | Open Source | `xf86-video-amdgpu`  |
+| ATI    | Open Source | `xf86-video-ati`     |
+| Intel  | Open Source | `xf86-video-intel`   |
+| NVIDIA | Open Source | `xf86-video-nouveau` |
 
-if [ -d /etc/X11/xinit/xinitrc.d ]; then
-  for f in /etc/X11/xinit/xinitrc.d/*; do
-    [ -x "$f" ] && . "$f"
-  done
-  unset f
-fi
+> Note: Some recommend not installing the `xf86-video-intel` driver for fourth
```

Comment:
> Note: Fourth generation intel users may want to use the default xorg driver,
> rather than installing `xf86-video-intel` driver package. For more
> information, see the [Arch wiki...

Is this better? I feel like it is worth mentioning. But a more neutral tone, sending them somewhere else is better.

Reply:
That sounds better to me.